src/buildfromsource/FSharp.Compiler.Private/FSComp.resx

```diff
@@ -4354,4 +4354,7 @@
   <data name="chkNoSpanLikeValueFromExpression" xml:space="preserve">
     <value>A Span or IsByRefLike value returned from the expression cannot be used at ths point. This is to ensure the address of the local value does not escape its scope.</value>
   </data>
+  <data name="tcTypeDoesNotInheritAttribute" xml:space="preserve">
+    <value>This type is not an attribute</value>
```

Comment:
I'd suggest changing it to:

"This type is not an attribute. To make it an attribute, you must derive from System.Attribute"

Reply:
I copied it from the C# error message. I don't think longer message is better.

Reply:
It was done correctly from the beginning in C# by having this restriction. We're now introducing a restriction that previously wasn't there, which I think justifies a suggestion in the error message.